src/views/Map.vue

```diff
@@ -372,13 +372,12 @@ export default {
 
     // get the current operator
     this.operator = this.getOperator;
-
-    if (getObjectValue(this.operator, "pacage") && !getObjectValue(this.operator, "title")) {
+    if (getObjectValue(this.operator, "numeroPacage") && !getObjectValue(this.operator, "title")) {
       alert(
         "Le numéro de Pacage n'est pas pour le moment rattaché à un opérateur." +
           "Merci de faire la mise à jour du numéro pacage de l'opérateur sur le site https://notification.agencebio.org/"
       );
-      this.operator.title = "pacage : " + this.operator.pacage;
+      this.operator.title = "pacage : " + this.operator.numeroPacage;
       this.operator.pacage = '"' + this.operator.pacage + '"';
```

Comment:
Ça devrait être `this.operator.numeroPacage` dans ce cas ?

Reply:
C'est pour le titre de la page. A la place du nom de l'exploitation vu qu'il n'y en a pas dans le cas ou on passe directement par un numéro pacage

Reply:
Cette ligne n'est plus utile, je la retire